### ISSUE TITLE
[GraphQL Basics course] [Main Task] [Soumaya]

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/playing-with-graphql#18]
https://github.com/GroceriStar/playing-with-graphql/invitations

Please start by reading the documentation, related to GraphQL: https://www.howtographql.com/
After reading each chapter - please briefly explain to me here what did you learn from it.
Format(example):

#### Chapter 1: Intro (https://graphql.org/learn/)
"It's covering the basics of graphql"
Questions: "How i can run it at my local PC"?

----

**Comment by atherdon** [comment]
Hi @bojack-horseman we're restarting our work on graphql server. buzz me if you want to continue

**Comment by bojack-horseman** [comment]
yeah sure, the only thing I haven't thoroughly worked with graphql before.

**Comment by atherdon** [comment]
Then let;s start from basics at this task(all data in description). when you read a chapter - explain here in short what you learn and when you'll complete learning all basics - i'll transit you into the real project

**Comment by bojack-horseman** [comment]
Alright Sure, I've started with the same and will try to post the 1st lesson summary by the weekend.

**Comment by atherdon** [comment]
Ok, cool will wait for it

>

**Comment by bojack-horseman** [comment]
Sorry about all the delays Arthur
Been caught up with multiple projects on the go.

**Comment by bojack-horseman** [comment]
Will try to get the summary part started with shortly.

**Comment by atherdon** [comment]
ok, cool. hope to move from learning this tech into coding a real service
and schemas

>
